src/popup/config/config.json

```diff
@@ -3,7 +3,7 @@
   "name": "Intune",
   "help": "https://intunewiki.com/wiki/Intune_Azure_Portal",
   "changelog": [
-	{
+    {
       "version": "4.6",
```

Comment:
Make sure to bump to `"version": "4.7",` so that the extension knows to use the updated config.

Reply:
Along with that, add a bit of text to the changelog to explain what `4.7` does.

Reply:
Really a good comment. Updated.